cmd/mist-api-connector/mist-api-connector.go

```diff
@@ -44,18 +44,20 @@ func main() {
 	noMistScrapeMetrics := fs.Bool("no-mist-scrape-metrics", false, "Scrape statistics from MistServer and publish to RabbitMQ")
 	sendAudio := fs.String("send-audio", "record", "when should we send audio?  {always|never|record}")
 	apiToken := fs.String("api-token", "", "Token of the Livepeer API to be used by the Mist server")
-	apiServer := fs.String("api-server", livepeer.ACServer, "Livepeer API server to use")
+	apiServer := fs.String("api-server", api.ProdServer, "Livepeer API server to use")
 	routePrefix := fs.String("route-prefix", "", "Prefix to be prepended to all created routes e.g. 'nyc-'")
 	playbackDomain := fs.String("playback-domain", "", "regex of domain to create routes for (ex: playback.livepeer.live)")
 	mistURL := fs.String("route-mist-url", "", "external URL of this Mist instance (used for routing) (ex: https://mist-server-0.livepeer.live)")
 	baseStreamName := fs.String("base-stream-name", "", "Base stream name to be used in wildcard-based routing scheme")
-	fEtcdEndpoints := fs.String("etcd-endpoints", "", "Comma-separated list of ETCD endpoints")
-	etcdCaCert := fs.String("etcd-cacert", "", "ETCD CA file name")
-	etcdCert := fs.String("etcd-cert", "", "ETCD client certificate file name")
-	etcdKey := fs.String("etcd-key", "", "ETCD client certificate key file name")
 	amqpUrl := fs.String("amqp-url", "", "RabbitMQ url")
 	ownRegion := fs.String("own-region", "", "Identifier of the region where the service is running, used for mapping external data back to current region")
 	_ = fs.String("config", "", "config file (optional)")
+	// Below are some deprecated flags.
+	// Keep them around for backward compatibility on deploys.
+	_ = fs.String("etcd-endpoints", "", "DEPRECATED")
+	_ = fs.String("etcd-cacert", "", "DEPRECATED")
+	_ = fs.String("etcd-cert", "", "DEPRECATED")
+	_ = fs.String("etcd-key", "", "DEPRECATED")
```

Comment:
I'm cool with this but we don't need them on deploys so I kind of doubt that anyone else does either